src/game/phase/phase_pause.h

```diff
@@ -3,3 +3,6 @@
 #include "game/phase/phase.h"
 
 extern PHASER g_PausePhaser;
+
+void Phase_Pause_Init(void);
+void Phase_Pause_Shutdown(void);
```

Comment:
This makes the outermost game shell code too reliant on very specific parts of the code, i.e. the pause phaser. As such I'd try to avoid introducing these new functions and instead move the requestor initiation and destruction to `Phase_Pause_Start` and `Phase_Pause_End`.